server/src/test/java/org/cloudfoundry/identity/uaa/zone/ZoneAwareClientSecretPolicyValidatorTests.java

```diff
@@ -39,6 +39,8 @@ void setUp() {
     @Test
```

Comment:
I really don't understand the commit message of this commit:

> allow empty secret int test with default policy because minlength is …
…0, only with strict it should fail

Also, if you could try to have a short title of <50 characters, and then you can add whatever description you want in the commit body.

Reply:
I committed first the test only
https://github.com/cloudfoundry/uaa/pull/2571/commits/66841d6e4ce6187ec6a06a547b7e9f4192526b92
Then the fix
https://github.com/cloudfoundry/uaa/pull/2571/commits/66841d6e4ce6187ec6a06a547b7e9f4192526b92

And then found out that the even the test was wrong
https://github.com/cloudfoundry/uaa/pull/2571/commits/66841d6e4ce6187ec6a06a547b7e9f4192526b92

Reply:
squashed now, close this here